Pipfile

```diff
@@ -1,20 +0,0 @@
-[[source]]
-url = "https://pypi.org/simple"
-verify_ssl = true
-name = "pypi"
-
-[dev-packages]
-pytest = "*"
-pytest-cov = "*"
-coveralls = "*"
```

Comment:
The package `coveralls` is used for tracking test coverage. Since you've removed it from here, you need to add this in `.travis.yml` for installation when the build starts.